config.json

```diff
@@ -13,7 +13,10 @@
   "homeChannel": 793285213823172628,
   "investigationsChannel": 745276933767430255,
   "warningDaysThreshold": 30,
-  "inviteExclusion": "microsoft",
+  "inviteExclusion": {
```

Comment:
This should be [] instead

Reply:
Fixed in https://github.com/Erisa/Cliptok/pull/35/commits/f416bd481da3254c8590dc3a8588f865c7496d1e